Cargo.toml

```diff
@@ -50,8 +46,8 @@ tokio = { version = "1.35.1", features = [
     "time",            # anables `tokio::time` namespace holding the `sleep` utility and `Duraction` struct
 ] }
 tokio-native-tls = { version = "0.3.1", optional = true }
-tokio-rustls = { version = "0.25.0", optional = true }
-rustls-native-certs = { version = "0.7.1", optional = true }
+tokio-rustls = { version = "0.26", optional = true }
+rustls-platform-verifier = { version = "0.4", optional = true }
```

Comment:
I've paged this out already (blame the holiday :sweat_smile:), but do we have types from either of these in our public API (esp. in trait implementations)? If so, this change will require a major version bump for faktory as well.

Reply:
Our `TlsStream` [derefs](https://docs.rs/faktory/latest/faktory/rustls/struct.TlsStream.html#impl-Deref-for-TlsStream%3CS%3E) to `tokio_rustls`'  stream. 

Besides, [`TlsStream::with_client_config`](https://docs.rs/faktory/latest/faktory/rustls/struct.TlsStream.html#method.with_client_config) accepts a config struct from `rustls`. We are using `rustls` re-exported from `tokio_rustls`, and a major [bump](https://github.com/rustls/tokio-rustls/compare/rustls:5343443...rustls:82b4a3e#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542R17) also took place there.

As for  `rustls-platform-verifier` - it's not part of out API.

Reply:
So this probably - and unfortunately - should be a `0.14.0` release candidate.

Reply:
Alternatively, we can  postpone these bumps and batch with other libs major versions updates that may come in the next couple of months.

I am now doing #61 and then will take #88. Bot of them are only extending what we currently got.

Reply:
Ah, that's unfortunate. Yeah, I think let's maybe hold off then in case we see other breaking changes coming down the line :+1: